bliss/models/location_encoder.py

```diff
@@ -305,87 +303,81 @@ def get_n_source_prior_log_prob(self, detection_rate):
         log_probs_last = torch.log1p(-torch.logsumexp(log_probs, 0).exp())
         return torch.cat((log_probs, log_probs_last.reshape(1)))
 
-    def encode_for_n_sources(self, var_params_flat, tile_n_sources):
-        """Get variational parameters conditioned on number of sources in tile.
+    def encode_for_n_sources_flat(self, var_params_flat: Tensor, tile_n_sources: Tensor):
+        """A wrapper the calls encode_for_n_sources.
+        Deprecated: it's better to call encode_for_n_sources directly,
+        and to always store tiles as a 2D array (i.e., not flattened).
```

Comment:
Ultimately I'd like to get rid of `encode_for_n_sources_flat` and `encode_for_n_sources`. In the meantime, by splitting the old `encode_for_n_sources` into two functions, we avoid branching based on the dimension of an input.

Reply:
I pushed a commit to get rid of this method

Reply:
To be clear, I fixed the underlying problem so we don't need two methods... I didn't revert the change or anything lol